components/form-toggle/index.js

```diff
@@ -21,27 +22,35 @@ class FormToggle extends Component {
 	}
 
 	render() {
-		const { className, checked, children, onChange = noop } = this.props;
-		const id = 'toggle-' + this.id;
+		const {
+			className,
+			checked,
+			onChange = noop,
+			showHint = true,
+			id = 'toggle-' + this.id,
```

Comment:
Ohhh, I missed that this was assigning the default. But why do we need a default? If the parent component isn't passing an ID, what purpose does the ID have?